.github/workflows/yocto-build-deploy.yml

```diff
@@ -0,0 +1,639 @@
+name: "Yocto"
+
+on:
+  workflow_call:
+    secrets:
+      BALENA_API_KEY:
+        description: balena API key for deploying releases
+        required: false
+      S3_ACCESS_KEY:
+        description: S3 access key
+        required: false
+      S3_SECRET_KEY:
+        description: S3 secret key
+        required: false
+      DOCKERHUB_USER:
```

Comment:
Let's remove the dockerhub credentials, we don't use them anymore and we won't backport the new build script to old releases anyway.

Reply:
We need these to pull the private `balena/balenaimg` from DockerHub.